### PR TITLE
Tweak ZJS dashboard

### PR DESCRIPTION
**Pull request by balloob**
## Proposed change

Simplify the Z-Wave JS dashboard

- Move diagnostic stuff to own card
- Move add devices to FAB to align with rest of HA

CleanShot 2021-12-02 at 23 19 22

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/config/integrations/integration-panels/zwave_js/zwave_js-config-dashboard.ts [hunk 1]
```diff
@@ -172,22 +176,56 @@ class ZWaveJSConfigDashboard extends LitElement {
                         )}
                       </mwc-button>
                     </a>
-                    <mwc-button @click=${this._addNodeClicked}>
+                  </div>
+                </ha-card>
+                <ha-card header="Diagnostics">
```
Comment (ludeeus) [inline src/panels/config/integrations/integration-panels/zwave_js/zwave_js-config-dashboard.ts:181]: The action items (other than dump) are not really diagnostics actions.
Do we even need a header for this card?

Comment (balloob) [reply src/panels/config/integrations/integration-panels/zwave_js/zwave_js-config-dashboard.ts:181]: Would you heal your network or re-configure the server if you're not in trouble?

Comment (ludeeus) [reply src/panels/config/integrations/integration-panels/zwave_js/zwave_js-config-dashboard.ts:181]: No, but that is more "Troubleshooting" or "Management" and not "Diagnostics"?

**Review by ludeeus — APPROVED** [review]

File: src/panels/config/integrations/integration-panels/zwave_js/zwave_js-config-dashboard.ts (hunk 1 above)
Comment (balloob) [reply src/panels/config/integrations/integration-panels/zwave_js/zwave_js-config-dashboard.ts:181]: Ah yes. 

@matthiasdebaat what do you think ?

Comment (bramkragten) [reply src/panels/config/integrations/integration-panels/zwave_js/zwave_js-config-dashboard.ts:181]: I think this is fine (at least for now)

**Pull request merged**